news/2022-02-25-skinning-contest-tides-of-winter-results.md

```diff
@@ -0,0 +1,67 @@
+
+![](/wiki/shared/news/2022-02-25-skinning-contest-tides-of-winter-results/ganyu.png)
+
+[Ganyu Artwork](https://www.pixiv.net/en/artworks/94717529) by [Doco](https://www.pixiv.net/en/users/71723824)
+
+## Winner - - 『Wintherest』 -
+
+Majestically crafted by the hands of a skinner extraordinaire, [Redo\_](https://osu.ppy.sh/users/7122165), [- 『Wintherest』 -](https://osu.ppy.sh/community/forums/topics/1498493) takes the lead with **2347** votes, surpassing the 2nd place by a short margin!
+
+Resorting to minimalist and modern styling methods, - 『Wintherest』 - brings the serenity of a wintry day and the vivid display of the aurorae together on your osu! client. The way snow is accumulated near the play button over which the logo of the skin is placed ensures a smooth transition to the anthracite layer that looks like a frosted glass filtering the lights of colour. On top of that, the use of white speech bubbles combined with various minuscule indicators strenghtens the skin's minimalistic ambience even further.
+
+Moreover, - 『Wintherest』 - not only keeps its minimalist and modern look consistent all throughout the skin, but gives the aurora effect further presence, which can be blatantly observed especially on the results screen. The gameplay interface, on the other hand, seems to be unable to keep itself away from this aurora effect, which was placed onto the health, combo and score indicators individually, as well as the usual minimalistic aura. Without a doubt, Redo\_ did it out of the park with this one, and surely the community agrees that their creation is the *Winterest* amongst all!
+
+<iframe width="100%" height="400" src="https://www.youtube-nocookie.com/embed/8_UWmCKCTfc" title="YouTube video player" frameborder="0" allow="accelerometer; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+
+With their - 『Wintherest』 - entry, Redo\_ gets the reward fit for a champion, a profile badge featuring the very logo of this winning masterpiece! Big congratulations to Redo\_!
+
+![](/wiki/shared/news/2022-02-25-skinning-contest-tides-of-winter-results/wintherest.png)
+
+---
+
+Be sure to have a look at the complete list of [Skinning Contest: Tides of Winter results](https://osu.ppy.sh/community/contests/137) to see how many votes your favourites gathered.
+
+It has been a great pleasure for us to host the sequel to the osu! Skinning Contest during this Winter, with additional ice cubes! We would like to thank the contestants for putting up with the challenge, and those following the contest till the very end for their endearing interest. If you have any positive or negative feedback for next iterations, feel free to share with us through the comment section below or via joining the [skinship Discord server](https://discord.skinship.xyz)!
```

Comment:
```suggestion
It has been a great pleasure for us to host the sequel to the osu! Skinning Contest during this winter, with additional ice cubes! We would like to thank the contestants for putting up with the challenge, and those following the contest till the very end for their endearing interest. If you have any positive or negative feedback for next iterations, feel free to share with us through the comment section below or via joining the [skinship Discord server](https://discord.skinship.xyz)!
```

Reply:
aren't season names capitalised? I will leave this for @Ephemeralis since i dont know enough about grammar rules

Reply:
> aren't season names capitalised? I will leave this for @Ephemeralis since i dont know enough about grammar rules

I checked with Merriam-Webster dictionary [Do the Names of the Seasons Get Capitalized?](https://www.merriam-webster.com/words-at-play/are-season-names-capitalized)

Reply:
only gets used as a proper noun when it's part of a title for a proper noun in this case, though i am guilty of this